docs/source/notes/cuda.rst

```diff
@@ -201,20 +201,42 @@ ensure proper synchronization.
 Stream semantics of backward passes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Internally, each backward CUDA op runs on the same stream that was used for its corresponding forward op.
-
-When manually supplying CUDA tensor(s) as a backward pass's initial gradient(s) (e.g.,
+A. Each backward CUDA op runs on the same stream that was used for its corresponding forward op.
+
+B. The stream semantics of a backward call with respect to surrounding ops are the same
+as for any other call. More concretely, when calling
+:func:`autograd.backward<torch.autograd.backward>`,
+:func:`autograd.grad<torch.autograd.grad>`, or
+:meth:`tensor.backward<torch.Tensor.backward>`,
+and optionally supplying CUDA tensor(s) as the  initial gradient(s) (e.g.,
 :func:`autograd.backward(..., grad_tensors=initial_grads)<torch.autograd.backward>`,
 :func:`autograd.grad(..., grad_outputs=initial_grads)<torch.autograd.grad>`, or
 :meth:`tensor.backward(..., gradient=initial_grad)<torch.Tensor.backward>`),
 the acts of
 
-1. populating the initial gradient(s) and
-2. invoking the backward pass
+1. optionally populating initial gradient(s),
+2. invoking the backward pass, and
+3. using the gradients
+
+have the same stream-semantics relationship as any group of ops::
+
+    # Safe, grads are used in the same stream context as backward()
+    with torch.cuda.stream(strm):
+        loss.backward()
+        use grads
+
+    # Unsafe
+    with torch.cuda.stream(strm):
+        loss.backward()
+    use grads
 
-have the same stream-semantics relationship as any pair of ops::
+    # Safe, with synchronization
+    with torch.cuda.stream(strm):
+        loss.backward()
+    torch.cuda.current_stream().wait_stream(strem)
```

Comment:
nit strem -> strm

Reply:
changed all "strm" to "s" to match other examples on the page.